bumble/device.py

```diff
@@ -3225,6 +3514,73 @@ async def indicate_subscriber(self, connection, attribute, value=None, force=Fal
     async def indicate_subscribers(self, attribute, value=None, force=False):
         await self.gatt_server.indicate_subscribers(attribute, value, force)
 
+    @host_event_handler
+    def on_advertising_set_termination(
+        self,
+        status,
+        advertising_handle,
+        connection_handle,
+        number_of_completed_extended_advertising_events,
+    ):
+        if not (
+            advertising_set := self.extended_advertising_sets.get(advertising_handle)
```

Comment:
The terminated set might be the lagacy one?

Reply:
fixed